01-Login/README.md

```diff
@@ -31,6 +31,15 @@ Otherwise:
 1. [Install Composer](https://getcomposer.org/doc/00-intro.md)
 1. [Run Composer](https://getcomposer.org/doc/01-basic-usage.md)
 
+### Installing Auth0 Plugin
```

Comment:
We can leave this whole section out if you change the Composer command above this from:

```
composer install
```

... to:

```
composer require auth0/login:"~5.0"
```

This will install all the packages and do the auto-discovery.

Reply:
Yup, we can go for it.